### ISSUE TITLE
The dataset name is not encoded in the image tile key

### ISSUE THREAD

**Issue by cgoina** [janelia-flyem/dvid#107]
Using the the kvautobus branch I created two dataset - 'testimagetile' and 'otherimagetile'. When I try to get the key for a tile with the same coordinates from each dataset I get the same key which could potentially clash when the tile data is persisted on scality.

```
http://localhost:8000/api/node/9522d12f15eb47f09c1cff765408355f/testimagetile/tilekey/xy/4/11_20_30
gives me
{"key": "0302000100000004038000001e800000148000000b”}

and 
http://localhost:8000/api/node/9522d12f15eb47f09c1cff765408355f/otherimagetile/tilekey/xy/4/11_20_30
gives me the same
{"key": "0302000100000004038000001e800000148000000b”}
```

**Comment by DocSavage** [comment]
The imagetile data type has a bug where the key used is just the type-specific key and not the full key (including version and instance id).  Fix and accompanying test will follow.

**Comment by DocSavage** [comment]
Fixed in commit 53e3c17b
